src/demo/article.html

```diff
@@ -6553,9 +6553,91 @@ <h2>
         References
       </h2>
 
-      <ol>
+      <ol itemtype="https://stencila.github.io/schema/CiteGroup">
```

Comment:
This isn't how `Cite` and `CiteGroup` are intended to be used. They are in-text links to other works listed in a `CreativeWork`'s `references` property. See the example at https://github.com/stencila/schema/blob/master/schema/CiteGroup.md

It's understandably confusing. We chose the name `Cite` rather than `Citation` to try and differentiate between the place that something is being referred to (a `Cite`) and the things that are being referred to (the `references`). Suggestions for how we can improve this welcome.